var/spack/repos/builtin/packages/freesurfer/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Freesurfer(Package):
+    """Freesurfer is an open source software suite for processing and analyzing
+    (human) brain MRI images."""
+
+    homepage = "http://freesurfer.net/"
+
+    # A license is required, but is free to obtain.
+    license_required = True
+    license_files = ["./license.txt"]
+
+    maintainers = ["robgics"]
+
+    def url_for_version(self, version):
```

Comment:
Please move this method so it occurs after the last `depends_on` directive.

Reply:
Changes made and committed